src/main/java/com/iexec/sms/tee/session/palaemon/PalaemonSessionService.java

```diff
@@ -252,20 +249,11 @@ private Map<String, Object> getApplicationComputeSecrets(TaskDescription taskDes
         tokens.put(IexecEnvUtils.IEXEC_APP_DEVELOPER_SECRET_PREFIX + secretIndex, appDeveloperSecret0);
 
         final HashMap<String, String> requesterSecrets = new HashMap<>();
-        Optional<TeeTaskComputeSecretCount> oMaxApplicationSecretIndex = teeTaskComputeSecretCountService.getMaxAppComputeSecretCount(applicationAddress, SecretOwnerRole.REQUESTER);
```

Comment:
Maybe we should add `@Deprecated` or a comment to the method teeTaskComputeSecretCountService.getMaxAppComputeSecretCount()` to explain why it is not used and that is intended to be remove in the future.

Reply:
I will see how do to it cleanly.
Globally, it should be all the `TeeTaskComputeSecretCountService` which becomes deprecated and could/should be deprecated.

Reply:
Everything has been removed